cmd/authctl/user/disable.go

```diff
@@ -0,0 +1,41 @@
+package user
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/internal/consts"
+	"github.com/ubuntu/authd/internal/proto/authd"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// disableCmd is a command to disable a user.
+var disableCmd = &cobra.Command{
+	Use:   "disable",
+	Short: "Disable a user managed by authd",
```

Comment:
I was wondering whether we should rename this to `lock` (and `enable` to `unlock`) for consistency with:
1. [systemd's JSON user records](https://systemd.io/USER_RECORD/), which have a `locked` boolean value to indicate whether the user may log in or not
2. the `org.freedesktop.Accounts` D-Bus API, which also uses "locked" with the same meaning as systemd.

On the one hand, I find "disabled" less ambiguous than "locked". On the other hand, it's nice to be consistent with other system services. @3v1n0 WDYT?

Reply:
Mhmhm... Yeah, I agree enabled and disabled are better to read, but I feel we've to sacrifice them for consistency sake as you say.

Reply:
I have renamed the commands to lock and unlock. Please check.

Reply:
Thanks @adombeck. I hope we can merge this soon. I look forward to adding more functionalities to the tool.